src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -907,7 +907,7 @@ public override ValueTask WriteAsync(ReadOnlyMemory<byte> buffer, CancellationTo
                     // TODO: until #9071 is fixed
                     if (http2Stream._abortException is OperationCanceledException)
                     {
-                        ExceptionDispatchInfo.Throw(http2Stream._abortException);
+                        throw new IOException(SR.net_http_request_aborted, http2Stream._abortException);
```

Comment:
This needs to be an OperationCanceledException:
https://github.com/dotnet/corefx/pull/39135/files#diff-8e043fcbb34ac4fb3db7952237aed9f0R908

Reply:
Since you submitted a PR, I'll back these changes out

Reply:
I'm surprised that tests didn't catch this. @wfurt?